.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/joshuatimmons/.local/share/virtualenvs/server-x8uVXr2X/bin/python"
```

Comment:
Oops, must not have added .vscode to .gitignore. I should be able to take care of this in my next pr.

Reply:
added .vscode to gitignore